bundles/af-core/src/test/resources/form/password/exporter-password.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "password-91417957c6",
+  "fieldType": "password",
+  "name": "password1732174214265",
+  "visible": true,
```

Comment:
We don't add the default values for many properties to keep the JSON payload minimal. Its only added if its part of JCR, as per the following code, https://github.com/adobe/aem-core-forms-components/blob/master/bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java#L241

Please verify this

Also, if it isn't working for other components fix the same.